tests/test_dcim.py

```diff
@@ -325,6 +325,28 @@ def test_get_all(self, mock):
             headers=HEADERS,
         )
 
+    @patch(
+        "pynetbox.core.query.requests.sessions.Session.get",
+        side_effect=[
+            Response(fixture="dcim/interface.json"),
+            Response(fixture="dcim/interface_trace.json"),
+            Response(fixture="dcim/cable.json"),
+            Response(fixture="dcim/cable.json"),
+            Response(fixture="dcim/cable.json"),
```

Comment:
This is the initial trace, and then the cable information being populated for each of the 3 hops in the trace.